samples/unit-tests/indicator-price-indicator/recalculations/demo.js

```diff
@@ -32,3 +50,47 @@ QUnit.test('Price indicator.', function (assert) {
         'No errors when points are missing.'
     );
 });
+
+QUnit.test('Datagrouping and setExtremes.', function (assert) {
+
+    var chart = Highcharts.stockChart('container', {
+            rangeSelector: {
+                selected: 5
+            },
+            series: [{
+                type: 'candlestick',
+                dataGrouping: {
+                    enabled: true
+                },
+                lastVisiblePrice: {
+                    enabled: true,
+                    label: {
+                        enabled: true
+                    }
+                },
+                lastPrice: {
+                    enabled: true,
+                    color: 'red'
+                },
+                data: getData()
+            }]
+        }),
+        series = chart.series[0];
+
+    assert.strictEqual(
+        series.points[series.points.length - 1].y === series.lastVisiblePrice.y,
+        true,
+        'Correct value when datagrouping enabled.'
```

Comment:
Using `Should ....` for test descriptions is preferred practice. Also it would be good if you could describe what the "correct value" is in this context.

Reply:
Done

Reply:
Since we're discussing assertion best practices - there is no point using `strictEqual` with a strict equal comparison in the first parameter and `true` in the second parameter. That is an `assert.ok` statement. Instead, write it like this: 

```js
assert.strictEqual(
        series.points[series.points.length - 1].y === series.lastVisiblePrice.y,
        true,
        'Indicator should show the close value of the last grouped point.'
);
```